src/renderer/services/db/schema.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @module db/schema
+ */
+
+// tslint:disable-next-line: import-name
+import BN from 'bn.js';
+
+// Buckets are separate database namespaces
+export enum Bucket {
+    // beacon chain
+    state, // hash -> BeaconState
+    attestation, // hash -> Attestation
+    block, // hash -> BeaconBlock
+    invalidBlock, // bad block
+    mainChain, // slot -> blockHash
+    chainInfo, // Key -> number64 | stateHash | blockHash
+    validator,
+    deposit, // index -> Deposit
+    exit, // hash -> VoluntaryExit
+    transfer, // hash -> Transfer
+    proposerSlashing, // hash -> ProposerSlashing
+    attesterSlashing // hash -> AttesterSlashing
+}
+
+export enum Key {
+    chainHeight,
+
+    latestState,
+    finalizedState,
+    justifiedState
```

Comment:
We can remove all Key values and keep it empty for now, until we start storing stuff